src/authservice/common/zarf.yaml

```diff
@@ -15,14 +15,3 @@ components:
         localPath: ../chart
         version: 1.0.1
         namespace: authservice
-    actions:
-      onDeploy:
-        after:
-          - description: Validate Authservice Package
-            maxTotalSeconds: 300
```

Comment:
One thing I noticed in testing this - the zarf behavior default seems to be waiting for 15m on resources (both the helm waits on "standard" resources and the kstatus waits). As this diff shows, previously we had a single wait set for 5 minutes on the Package CR only.

I haven't yet identified if we can override the default 15m wait but it might be worth evaluating whether we would want to. The default is probably fine for now - just means a longer wait time for failure potentially?

Reply:
Just checked with zarf folks - sounds like the timeout is a CLI option so end users could override this when deploying. Probably can ignore this comment for now!

Reply:
Yes, I agree. It seems the on failure cases we'll be waiting a bit longer. Let's see if this will require any adjustments in the future.